rackcity/views/it_model_views.py

```diff
@@ -48,6 +49,46 @@ def model_add(request):
     )
 
 
+@api_view(['POST'])
+@permission_classes([IsAdminUser])
+def model_modify(request):
+    """
+    Modify an existing model
+    """
+    data = JSONParser().parse(request)
+    failure_message = ""
+    if 'id' not in data:
+        failure_message += "Must include id when modifying a model. "
+    else:
+        id = data['id']
+        try:
+            ITModel.objects.get(id=id)
+        except ObjectDoesNotExist:
+            failure_message += "No existing model with id="+str(id)+". "
+        else:
+            serializer = ITModelSerializer(data=data)
+            if not serializer.is_valid(raise_exception=False):
```

Comment:
This is going to fail if the required model objects are not included (vendor, model number, height). We could either make a new Serializer for which all fields are not required (personally I think this is the better option), or we could implement deserialization ourselves just checking each `serializer.data[field] is not None` and then validating the value of each...!